.github/workflows/image-build-source.yml

```
@@ -0,0 +1,69 @@
+## Github workflow to build a docker image from source
+
+name: Docker Image (Source)
+
+on:
+  workflow_dispatch:
+  workflow_call:
+
+defaults:
+  run:
+    shell: bash
+
+## Define which docker arch to build for
+env:
+  docker_platforms: linux/amd64
+  docker-org: blockstack
+
+concurrency:
+  group: docker-image-source-${{ github.head_ref || github.ref || github.run_id }}
+  ## Always cancel duplicate jobs
+  cancel-in-progress: true
+
+jobs:
+  ## Runs anytime `ci.yml` runs or when manually called
+  image:
+    name: Build Image
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      ## Build a maximum of 2 images concurrently based on matrix.dist
+      max-parallel: 2
+      matrix:
+        dist:
+          - debian
+    steps:
+      ## Setup Docker for the builds
+      - name: Docker setup
+        uses: stacks-network/actions/docker@main
+        with:
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+
+      ## Set docker metatdata
+      - name: Docker Metadata ( ${{matrix.dist}} )
+        id: docker_metadata
+        uses: docker/metadata-action@96383f45573cb7f253c731d3b3ab81c87ef81934 #v5.0.0
+        with:
+          images: |
+            ${{env.docker-org}}/${{ github.event.repository.name }}
+            ${{env.docker-org}}/stacks-core
```

Same here, I believe this will result in a list of:
```
stacks-network/stacks-core
stacks-network/stacks-core
```

Instead, did you intend this to continue pushing to the old Docker repo?

```suggestion
            ${{env.docker-org}}/${{ github.event.repository.name }}
            ${{env.docker-org}}/stacks-blockchain
```